components/org.wso2.carbon.identity.recovery/src/main/java/org/wso2/carbon/identity/recovery/RecoveryScenarios.java

```diff
@@ -1,15 +1,30 @@
-//package org.wso2.carbon.identity.recovery;
-//
-///**
-// * Recovery Scenarios Constants.
-// */
-//public enum RecoveryScenarios {
-//    NOTIFICATION_BASED_PW_RECOVERY,
-//    QUESTION_BASED_PWD_RECOVERY,
-//    USERNAME_RECOVERY,
-//    SELF_SIGN_UP,
-//    ASK_PASSWORD,
-//    ADMIN_FORCED_PASSWORD_RESET_VIA_EMAIL_LINK,
-//    ADMIN_FORCED_PASSWORD_RESET_VIA_OTP,
-//
-//}
+/*
+* Copyright (c) 2017, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+* http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+package org.wso2.carbon.identity.recovery;
+
+/**
+ * Recovery Scenarios Constants.
+ */
+public enum RecoveryScenarios {
+    NOTIFICATION_BASED_PW_RECOVERY,
+    QUESTION_BASED_PWD_RECOVERY,
```

Comment:
Better to maintain consistency, ie. either let's user PWD == 'Password' or PW == 'Passoword'
IMO let's use PASSWORD without going for PWD or PW